akka-docs/src/main/paradox/discovery/index.md

```diff
@@ -0,0 +1,209 @@
+# Discovery
+
+@@@ warning
+
+This module is currently marked as @ref:[may change](../common/may-change.md)
+This means that API or semantics can
+change without warning or deprecation period and it is not recommended to use
+this module in production.
```

Comment:
Can't we remove that part of not for production? We know this is and will be heavily used in production. A module can be marked may change but still be ready for for production.